config/crowdtangle_list.json

```diff
@@ -1,3 +0,0 @@
-{
-	"crowdtangle_list_account_pairs":{}
-}
```

Comment:
Let's add this file to .gitignore for when it gets created

Reply:
done